docs/guides/distribution/updater.md

```diff
@@ -2,220 +2,126 @@
 sidebar_position: 5
 ---
 
-# Updater
-
-## Configuration
-
-Once you have your Tauri project ready, you need to configure the updater.
-
-Add this in tauri.conf.json
-
-```json
-"updater": {
-    "active": true,
-    "endpoints": [
-        "https://releases.myapp.com/{{target}}/{{current_version}}"
-    ],
-    "dialog": true,
-    "pubkey": "YOUR_UPDATER_SIGNATURE_PUBKEY_HERE"
-}
-```
-
-The required keys are "active", "endpoints" and "pubkey"; others are optional.
-
-"active" must be a boolean. By default, it's set to false.
+import Command from '@theme/Command'
 
-"endpoints" must be an array. The string `{{target}}` and `{{current_version}}` are automatically replaced in the URL allowing you to determine [server-side](#update-server-json-format) if an update is available. If multiple endpoints are specified, the updater will fallback if a server is not responding within the pre-defined timeout.
-
-"dialog" if present must be a boolean. By default, it's set to true. If enabled, [events](#events) are turned off as the updater handles everything. If you need the custom events, you MUST turn off the built-in dialog.
-
-"pubkey" must be a valid public-key generated with Tauri CLI. See [Signing updates](#signing-updates).
+# Updater
 
-### Update Requests
+Tauri offers a built-in updater for the MSI (Windows), AppImage (Linux) and App bundle (macOS) distribution formats.
 
-Tauri is indifferent to the request the client application provides for update checking.
+## Configuration
```

Comment:
I don't think this heading is needed at all especially since there is another one further down called "Tauri Configuration". I think the heading could be removed but keep the content under it.

Reply:
Yeah you're right. This heading is indeed a bit much.